docs/user-documentation/glossary.md

```diff
@@ -0,0 +1,79 @@
+The following glossary of terms addresses an Islandora 8 context. When comparing new Islandora and Fedora to older versions it may also be helpful to reference [the Islandora 7 Glossary](https://wiki.duraspace.org/display/ISLANDORA/APPENDIX+E+-+Glossary).
+ 
+## CLAW
+ CLAW (CLAW Linked Asset WebDataFrameWork) was the production code name for the software released in June, 2019 as _Islandora 8_
+ 
+## Collection
+Collections are a way of grouping related objects together in Islandora. They function much like directories on a computer; a collection can “contain” any number of related resource nodes and sub-collections. 
+ 
+## Drupal 
+Drupal is an open source content management system (CMS) written in PHP. Known for being extremely flexible and extensible, Drupal is supported by a community of over 630,000 users and developers. Drupal sites can be customized and themed in a wide variety of ways.
+ 
+Drupal functions as the _front-end_ of Islandora.
+ 
+## Drupal Core
+The files, themes, profiles, and modules included with the standard project software download.
+ 
+## Drupal Roles
+Roles are a way of assigning specific permissions to a group of users. Any user assigned to a role will have the same permissions as all other users assigned to that role. This allows you to control which users have permission to view, edit, or delete content in both Drupal and the Fedora repository.
```

Comment:
> This allows you to control which users have permission to view, edit, or delete content in both Drupal and the Fedora repository.

The Roles allow you to control what actions members of that role can perform in Drupal; however, it is membership in the fedoraAdmin role that causes those actions to be reflected in Fedora. (E.g. you can add me to a role that allows me to create resource nodes; but unless I'm a member of fedoraAdmin that new node won't be indexed.) The fedoraAdmin role doesn't technically need any permissions to serve its purpose.